README.md

```diff
@@ -108,6 +108,12 @@ resilientRawRepresentableProperty
 
 **Note:** One difference the errors available on the property wrapper and those reported to the `ResilientDecodingErrorReporter`, is the latter _does not_ report `UnknownNovelValueError`s by default (`UnknownNovelValueError` is thrown when a non-frozen `ResilientRawRepresentable`'s `init(rawValue:)` returns `nil`). You can alter this behavior by calling `errors(includeUnknownNovelValueErrors: true)` on the error digest. 
 
+## Frequently Asked Questions
+
+### Will `Resilient` work with generic type arguments?
+
+No. If you have a type that is generic over `<T>` and specify `Resilient<T>`  it will not matter if `T` is an array or dictionary, it will be treated as a single value.
```

Comment:
Maybe `SomeResilient<T>` to make it more clear you aren’t referring to a type provided by this library?